packages/auth/src/interaction/routes.ts

```diff
@@ -148,6 +154,7 @@ async function getGrantDetails(
   ctx.body = {
     grantId: interaction.grant.id,
     access: access.map(toOpenPaymentsAccess),
+    subjects: subjects.map(toOpenPaymentsSubject),
```

Comment:
there can only be one subject for a grant, but the `sub_ids` can be many. So this can either be a `subject` object with `sub_ids` array, (like the Open Payments spec), or we can directly return `sub_ids` here.

Reply:
The same applies for the subject updates in the GraphQL schema